src/DeferredFutures.jl

```diff
@@ -53,6 +54,8 @@ end
 Serialize a DeferredFuture such that it can de deserialized by `deserialize` in a cluster.
 """
 function Base.serialize(s::AbstractSerializer, ref::DeferredFuture)
+    Base.Serializer.serialize_cycle(s, ref) && return
```

Comment:
I'm not sure how to test that without adding Dispatcher.jl as a dependency for tests, which might be a good idea. I'll try that tomorrow.